GChan/ImageLink.cs

```diff
@@ -39,47 +39,55 @@ public string GenerateNewFilename(ImageFileNameFormat format)
 

```

Comment:
The code above here and code below that that splits on the url and stuff to `Path.GetFileNameWithoutExtension(URL)` to get the file id is not graceful and is actually not needed anymore since we store the file id in the `Tim` property. Use that instead.